BLocal.Web.Manager.MVC3/Views/Home/Index.cshtml

```diff
@@ -2,17 +2,57 @@
 @{
     ViewBag.Title = "Authentication";
 }
+<meta name="google-signin-client_id" content="@ConfigurationManager.AppSettings["GoogleOAuthCliendId"]">
+<script src="https://apis.google.com/js/platform.js" async defer></script>
+<script type="text/javascript">
+    function onSignIn(googleUser) {
+        // Send ID-Token to controller
+        $.ajax({
+            url: '@Url.Action("GoogleLogin", "Home")',
+            type: 'POST',
+            dataType: 'json',
+            cache: false,
+            data: { token: googleUser.getAuthResponse().id_token },
+            success: function (data) {
+                if (data === "ok") {
+                    location.href = "/Home/Overview";
+                } else if (data === "bad_domain") {
+                    alert("Your google account's domain is not allowed to login using google sign in. You will be logged out from current account. Please try to log in using account allowed to enter.");
```

Comment:
Please revise the statement - it should state somehting like Your ... is not allowed to use this service.